caffe2/ideep/operators/utility_ops.cc

```diff
@@ -25,6 +25,23 @@ class CopyCPUToIDEEPOp final : public IDEEPOperator {
   }
 };
 
+class IDEEPCopyOp final : public IDEEPOperator {
+ public:
+  USE_SIMPLE_IDEEP_CTOR_DTOR(IDEEPCopyOp);
+  USE_IDEEP_DEF_ALIASES();
+
+  bool RunOnDevice() override {
+    const auto& X = OperatorBase::Input<itensor>(0);
+    auto* Y = Output(0);
+    if (X != *Y) {
+      Y->reinit_like(X);
+    }
+    ideep::direct_copy::compute(X, *Y);
```

Comment:
Move this line into the if condition?

Reply:
it is a op which used in data_parallel_model.py. It support multi-devices mode.

Reply:
yes,it should be moved to if condition.

Reply:
This op is need by multi-devices mode for mkl-dnn,if we fallback this op,the performance will be affected。